hapi/hedera-protobufs/services/consensus_update_topic.proto

```diff
@@ -87,4 +88,19 @@ message ConsensusUpdateTopicTransactionBody {
      * If unspecified, no change.
      */
     AccountID autoRenewAccount = 9;
+
+    /**
+     * Access control for update/delete of custom fees. Null if there is no key.
+     */
+    Key fee_schedule_key = 10;
+
+    /**
+      * If the transaction contains a signer from this list, no custom fees are applied.
+      */
+    repeated Key free_messages_key_list = 11;
+
+    /*
+     * The custom fee to be assessed during a message submission to this topic
+     */
+    repeated ConsensusCustomFee custom_fees = 12;
```

Comment:
Text should specify behavior of the field and follow correct formatting:
```suggestion

    /**
     * Access control for update/delete of custom fees.
     * <p>
     * If this field is unset, the current value SHALL NOT change.<br/>
     * If the `fee_schedule_key` is not set in state,
     * this field MUST NOT be set.<br/>
     * If this field is set, the existing key MUST sign this transaction.<br/>
     * If this field is set, the new key MUST sign this transaction.
     */
    Key fee_schedule_key = 10;

    /**
     * A list of "privileged payer" keys.
     * <p>
     * If a submit transaction is signed by _any_ key from this list,
     * custom fees SHALL NOT be charged for that transaction.<br/>
     * This list SHALL _implicitly_ contain the topic `admin_key`,
     * the topic `submit_key`, and the topic `fee_schedule_key`, if any
     * of those keys is set.
     */
    repeated Key free_messages_key_list = 11;

    /**
     * A set of custom fee definitions.<br/>
     * These are fees to be assessed for each submit to this topic.
     * <p>
     * If this list is empty, the current set of custom fees
     * SHALL NOT change.<br/>
     * If this list is not empty, the current `fee_schedule_key` MUST sign
     * this transaction.<br/>
     * If this list is not empty, each fee defined in this set SHALL
     * be evaluated for each message submitted to this topic, and the
     * resultant total assessed fees SHALL be charged.<br/>
     * If this list is not empty, custom fees defined here SHALL be
     * charged _in addition to_ the base network and node fees.
     */
    repeated ConsensusCustomFee custom_fees = 12;
```
Note: I am guessing a bit as to behavior in this suggestion; please correct any errors above.

Reply:
Fixed.